<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>699</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Reaper</string>
  </property>
  <property name="statusTip">
   <string>Visit http://reaper.social for help &amp; tutorials</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="statusTip">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="introTab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>0. Introduction</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_19">
            <property name="font">
             <font>
              <pointsize>28</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Welcome to Reaper!</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_20">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_41">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Â© The University of Queensland&lt;/p&gt;&lt;p&gt;Developed by &lt;a href=&quot;https://github.com/ScriptSmith&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Adam Smith&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="label_22">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Reaper helps you gather social media data, no coding required.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_38">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_23">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Simply:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_24">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>1. Add &amp; verify your API Keys</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_25">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>2. Input your parameters</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_26">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>3. Add your scraping instruction to the queue</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_27">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>4. Download your data</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="keyTab">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>1. API Keys</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_42">
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>By using these platforms, you are agreeing to their terms &amp; conditions</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QAbstractScrollArea
{
background-color: transparent;
}
QWidget#scrollAreaWidgetContents{
background-color: transparent; /*or a colour*/
} </string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1038</width>
             <height>575</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sourceTab">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>2. Sources</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Select the source that you want to scrape from</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Double-click a node to see and select its edges</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="sourcesTabs">
          <property name="tabPosition">
           <enum>QTabWidget::North</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="elideMode">
           <enum>Qt::ElideNone</enum>
          </property>
          <property name="usesScrollButtons">
           <bool>true</bool>
          </property>
          <property name="tabsClosable">
           <bool>false</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="queueTab">
       <attribute name="title">
        <string>3. Job Queue</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="queueLayout">
          <item>
           <layout class="QVBoxLayout" name="queueTools">
            <item>
             <widget class="QToolButton" name="queueUp">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>up.png</normaloff>up.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="queueDown">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>down.png</normaloff>down.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="queueRemove">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>remove.png</normaloff>remove.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>Queue Controls</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="queueStart">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="queueStop">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="queueClear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="queueSave">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Save Queue to file</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="progressTab">
       <attribute name="title">
        <string>4. Current Job</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="progressLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1082</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Open</string>
     </property>
     <widget class="QMenu" name="menuBackups">
      <property name="title">
       <string>Backups</string>
      </property>
      <addaction name="actionQueue"/>
      <addaction name="actionJob"/>
     </widget>
     <addaction name="actionAPI_Key_file"/>
     <addaction name="actionJob_Queue"/>
     <addaction name="actionScraping_state"/>
     <addaction name="separator"/>
     <addaction name="menuBackups"/>
    </widget>
    <widget class="QMenu" name="menuSave">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Save</string>
     </property>
     <addaction name="actionAPI_Keys"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="menuSave"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionReport_a_bug"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionLicenses"/>
    <addaction name="separator"/>
    <addaction name="actionWebsite"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionHistory_2"/>
    <addaction name="actionAPI_Key_Database"/>
    <addaction name="separator"/>
    <addaction name="actionAdvanced_mode"/>
    <addaction name="actionDark_mode"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAPI_Key_file">
   <property name="text">
    <string>API Keys</string>
   </property>
  </action>
  <action name="actionScraping_state">
   <property name="text">
    <string>Scraping state</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Reaper</string>
   </property>
  </action>
  <action name="actionLicenses">
   <property name="text">
    <string>Licenses</string>
   </property>
  </action>
  <action name="actionAPI_Keys">
   <property name="text">
    <string>API Keys</string>
   </property>
  </action>
  <action name="actionJob_Queue">
   <property name="text">
    <string>Job Queue</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionReport_a_bug">
   <property name="text">
    <string>Report a bug</string>
   </property>
  </action>
  <action name="actionScraping">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scraping</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionHistory_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionAPI_Key_Database">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>API Key Database</string>
   </property>
  </action>
  <action name="actionAdvanced_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Advanced mode</string>
   </property>
  </action>
  <action name="actionWebsite">
   <property name="text">
    <string>Website</string>
   </property>
  </action>
  <action name="actionQueue">
   <property name="text">
    <string>Queue</string>
   </property>
  </action>
  <action name="actionJob">
   <property name="text">
    <string>Job</string>
   </property>
  </action>
  <action name="actionDark_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark mode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
